docs/index.html

```diff
@@ -126,29 +130,39 @@ <h3>Share your projects with your team</h3>
 	<script>
 		var version = '1.2.0';
 
-		function getFileName() {
+		function getIsMacOS() {
```

Comment:
```suggestion
		function isMacOS() {
```

by convention i always use this naming (it's just a suggestion) :
- getXXX() for a value
- isXXX() for a boolean